ios/Magnet/Info.plist

```diff
@@ -26,21 +26,17 @@
 	<dict>
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
```

Comment:
Interestingly it appears as though `NSAllowsAbititraryLoads` was set to `true` all along. Although our info.plist was malformed, in that it had two `<key>NSAppTransportSecurity</key>` sections.